jinja2/base.html

```diff
@@ -70,6 +70,17 @@
 
   {{ soapbox_messages(get_soapbox_messages(request.path)) }}
 
+  {% if user.is_authenticated() %}
+      {% if request.session.sociallogin_provider == 'persona' %}
+          <div class="notification warning" data-level="warning">
+              {% trans persona_info='/docs/MDN/Contribute/Persona_Sign_In_on_MDN' %}
```

Comment:
The URL changed to `/docs/MDN/Contribute/Persona_sign-in`. This might be a one-time change, or they may keep adjusting. It may be appropriate as a Constance variable, but we'd need a backend change to pass the variable to the template. Redirects are... OK.